src/main/resources/application.yml

```
@@ -19,3 +19,7 @@ spring:
 
 mybatis:
   mapper-locations: classpath:mappers/**/*.xml
+
+mail:
+  address: ksh251024@naver.com  # 발신자 메일
+  password:                     # 메일 비번
```

yamllint

[warning] too many spaces after colon

(colons)